server/cli/accounting.go

```diff
@@ -18,9 +18,11 @@
 package cli
```

Comment:
Separately, it probably also makes sense to move `cli` up to a top-level directory since it contains a mixture of client and server functionality.